class-plugin-autoupdate-filter.php

```diff
@@ -199,6 +248,27 @@ function() use ( $slug ) {
 		}
 	}
 
+	/**
+	 * Big admin warning on plugins page if we've engaged the killswitch
+	 *
+	 */
+	public function killswitch_engaged_admin_warning() {
+		if ( ! isset ( $this->settings['team51_autoupdate_settings_disable_all_toggle'] ) ) {
+			return;
+		}
+		// add notice to the top of the screen
+		global $pagenow;
+		if ( 'plugins.php' === $pagenow && 'on' === $this->settings['team51_autoupdate_settings_disable_all_toggle'] ) {
+			add_action(
+				'admin_notices',
+				function() {
+					echo '<div class="error"><p><strong style="color:red;"> Caution:</strong> All autoupdates are currently deactivated. Please contact the WordPress Special Projects team before manually updating.</p></div>';
```

Comment:
Shouldn't the classes on this notice be `notice notice-error`? https://developer.wordpress.org/reference/hooks/admin_notices/

Reply:
Yes! Fixed.